### PR TITLE
Change HLS to HTTP Streaming

### PR DESCRIPTION
**Pull request by justindotpub**
- Change HTTP Live Streaming/HLS to "HTTP Streaming", and link to RFC for HTTP/2 Streams.
- Update other links for consistency

### CODE REVIEW

File: car-pool/README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ We propose developing a transport agnostic CAR-based synchronization mechanism,
 
 This proposal resolves multiple major, persistent challenges that numerous production IPFS operators have experienced. We believe the scope of this work to be an important stepping stone in the maturation of IPFS and Filecoin in production settings.
 
-We propose extending the existing efforts to move specifically CAR files over HTTP (e.g. Filecoin point-to-point CAR, Qri’s DSync), and enable other transports such as WSS, HLS, and WebRTC. We know of several other projects in the broader ecosystem that would like to make use of this protocol, from storage providers and CDNs to decentralized social media protocols.
+We propose extending the existing efforts to move specifically CAR files over HTTP (e.g. Filecoin point-to-point CAR, Qri’s DSync), and enable other transports such as [WSS](https://datatracker.ietf.org/doc/html/rfc6455), [HTTP Streaming](https://datatracker.ietf.org/doc/html/rfc2616#section-3.6.1), and [WebRTC](https://datatracker.ietf.org/doc/html/rfc8835). We know of several other projects in the broader ecosystem that would like to make use of this protocol, from storage providers and CDNs to decentralized social media protocols.
```
Comment (justindotpub) [inline car-pool/README.md]: I didn't need to add the other links, but it felt weird not to after doing so for HTTP Streaming.  I chose to keep just HTTP Streaming vs adding verbiage around how we really mean chunked transfer-encoding.

Comment (expede) [reply car-pool/README.md]: Links never hurt! Thanks 😄

**Comment by expede** [comment]
@justincjohnson it looks like we need to define which version of HTTP, because this is a meaningful difference between the HTTP/1.1 and HTTP/2:

From the [HTTP/2 spec](https://www.rfc-editor.org/rfc/rfc7540):

> HTTP/2 uses DATA frames to carry message payloads. The chunked transfer encoding defined in [Section 4.1](https://httpwg.org/specs/rfc7230.html#chunked.encoding) of [[RFC7230]](https://httpwg.org/specs/rfc7540.html#RFC7230) MUST NOT be used in HTTP/2.

My gut says that HTTP/1.1 is supported everywhere, so starting there is a good idea, but we may want to (eventually) break the two streaming options out into their own specs so that we can enable chunked and data frames.

**Comment by expede** [comment]
Screenshot 2023-01-18 at 19 46 19

[Source](https://caniuse.com/?search=http%2F2)

Hmm yeah, we may actually just want to jump directly to HTTP/2 🤔

**Comment by expede** [comment]
I just double checked, and the version that I had previously used that I found straightforward was in fact HTTP/2

**Comment by justindotpub** [comment]
This makes perfect sense to me and is honestly where I preferred we land.  I've updated the previous links and added a line explicitly stating HTTP/2 is required.  Thank you.

**Review by expede — APPROVED** [review]
LGTM! 🚀 🎉

**Pull request merged**